src/test/skript/tests/syntaxes/conditions/CondIsInvulnerable.sk

```diff
@@ -0,0 +1,3 @@
+test "CondIsInvulnerable":
+  assert creative is invulnerable with "Creative mode should be invulnerable"
+  assert survival is not invulnerable with "Survival mode should not be invulnerable"
```

Comment:
```suggestion
  assert survival is not invulnerable with "Survival mode should not be invulnerable"

```
this should also test the cases for entities imo, and the cases where the input is null (`{_}`)

Reply:
you also need a `running minecraft "1.20.6"` condition

Reply:
you can actually use the method exists condition!